packages/vega-typings/package.json

```diff
@@ -21,8 +21,8 @@
   "dependencies": {
     "@types/geojson": "7946.0.4",
     "vega-event-selector": "^3.0.1",
-    "vega-expression": "^5.0.1",
-    "vega-util": "^1.17.1"
+    "vega-expression": "^5.1.0",
```

Comment:
did you mean to version these as well?

Reply:
Yeah, I think otherwise this leads to issues like #3771